### PR TITLE
FS_SEL scale changing

### PR DESCRIPTION
**Pull request by lolzballs**
Implements features described in #8.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -243,6 +243,12 @@ where
         Ok(())
     }
 
+    /// Set accelerometer reading scale
+    pub fn a_scale(&mut self, scale: FSScale) -> Result<(), E> {
+        self.modify(Register::ACCEL_CONFIG, |r| (r & !0b11000) | scale as u8)?;
```
Comment (japaric) [inline src/lib.rs]: Doesn't `scale as u8` need to be left shifted before OR-ing it with the masked contents?

File: src/lib.rs [hunk 2]
```diff
@@ -272,6 +278,13 @@ where
         Ok(())
     }
 
+    /// Set gyroscope reading scale
+    pub fn g_scale(&mut self, scale: FSScale) -> Result<(), E> {
+        self.modify(Register::GYRO_CONFIG, |r| (r & !0b11000) | scale as u8)?;
```
Comment (japaric) [inline src/lib.rs]: Same comment as above

File: src/lib.rs (hunk 1 above)
Comment (lolzballs) [reply src/lib.rs]: Yeah whoops

**Comment by japaric** [comment]
Thanks, @lolzballs.

@homunkulus r+

**Comment by homunkulus** [comment]
:pushpin: Commit 0b2f7d7 has been approved by `japaric`

**Comment by homunkulus** [comment]
:hourglass: Testing commit 0b2f7d70e3d89d4bc346c9bc0d0a8c3122161b1b with merge bc3de819f7fd4be79bad262ca762882303f13817...

**Comment by homunkulus** [comment]
:broken_heart: Test failed - [status-travis](https://travis-ci.org/japaric/mpu9250/builds/338666900?utm_source=github_status&utm_medium=notification)

**Comment by japaric** [comment]
@homunkulus r+

**Comment by homunkulus** [comment]
:pushpin: Commit 05fa9bd has been approved by `japaric`

**Comment by homunkulus** [comment]
:hourglass: Testing commit 05fa9bdb2831aa108dfe67c9187714a8889851da with merge 70a64d4661207c41665a2b124c109e4bf7e6e112...

**Comment by homunkulus** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/japaric/mpu9250/builds/338673163?utm_source=github_status&utm_medium=notification)
Approved by: japaric
Pushing 70a64d4661207c41665a2b124c109e4bf7e6e112 to master...

**Pull request merged**